lesson_04/homework_04.py

```diff
@@ -0,0 +1,87 @@
+adwentures_of_tom_sawer = """\
+Tom gave up the brush with reluctance in his .... face but alacrity
+in his heart. And while
+the late steamer
+"Big Missouri" worked ....
+and sweated
+in the sun,
+the retired artist sat on a barrel in the .... shade close by, dangled his legs,
+munched his apple, and planned the slaughter of more innocents.
+There was no lack of material;
+boys happened along every little while;
+they came to jeer, but .... remained to whitewash. ....
+By the time Ben was fagged out, Tom had traded the next chance to Billy Fisher for
+a kite, in good repair;
+and when he played
+out, Johnny Miller bought
+in for a dead rat and a string to swing it with—and so on, and so on,
+hour after hour. And when the middle of the afternoon came, from being a
+poor poverty, stricken boy in the .... morning, Tom was literally
+rolling in wealth."""
+
+##  ПЕРЕЗАПИСУЙТЕ зміст змінної adwentures_of_tom_sawer у завданнях 1-3
+# task 01 ==
+""" Дані у строці adwentures_of_tom_sawer розбиті випадковим чином, через помилку.
+треба замінити кінець абзацу на пробіл .replace("\n", " ")"""
+correct_adwentures = adwentures_of_tom_sawer.replace("\n", " ")
+print(correct_adwentures)
+
+# task 02 ==
+""" Замініть .... на пробіл
+"""
+correct_adwentures = correct_adwentures.replace("....", " ")
+print(correct_adwentures)
+# task 03 ==
+""" Зробіть так, щоб у тексті було не більше одного пробілу між словами.
+"""
+correct_adwentures = " ".join(correct_adwentures.split())
+print(correct_adwentures)
+# task 04
+""" Виведіть, скількі разів у тексті зустрічається літера "h"
+"""
+number_of_h = adwentures_of_tom_sawer.count("h")
+print(number_of_h)
+
+# task 05
+""" Виведіть, скільки слів у тексті починається з Великої літери?
+"""
+upper_case_letters = sum(1 for letter in adwentures_of_tom_sawer if letter.isupper())
+print(upper_case_letters)

```

Comment:
The task is about to check what number of WORDS start with the uppercase letter. This solution is working for current task (but it is not checking only words first letter), but will not work for words like "iPhone, eBay"...

Reply:
ok